packages/dcos-integration-test/extra/test_ec2.py

```diff
@@ -145,7 +82,17 @@ def test_move_external_volume_to_new_agent(cluster):
         cluster.deploy_marathon_app(read_app, **deploy_kwargs)
         cluster.destroy_marathon_app(read_app['id'])
     finally:
-        try:
-            _delete_ec2_volume(test_label)
-        except Exception as ex:
-            raise Exception("Failed to clean up volume {}: {}".format(test_label, ex)) from ex
```

Comment:
All this is doing is raising a generic exception and attaching test_label. Rather, I just log this to output before delete action runs and let the exception raise itself. It is functionally identical and a little easier to read IMO. Also, I remembered @jgehrcke specifically pointing out this line of code in a JIRA as lacking, so I figured I would make his day ;)